src/lsm/lsm_tree.c

```diff
@@ -851,45 +851,6 @@ __wt_lsm_tree_retire_chunks(WT_SESSION_IMPL *session,
 }
 
 /*
- * __wt_lsm_tree_alter --
- *	Alter an LSM tree.
- */
-int
-__wt_lsm_tree_alter(
-    WT_SESSION_IMPL *session, const char *uri, const char *cfg[])
-{
-	WT_DECL_RET;
-	WT_LSM_CHUNK *chunk;
-	WT_LSM_TREE *lsm_tree;
-	u_int i;
-	bool locked;
-
-	locked = false;
-
-	/* Get the LSM tree. */
-	WT_RET(__wt_lsm_tree_get(session, uri, false, &lsm_tree));
-
-	/* Prevent any new opens. */
-	__wt_lsm_tree_writelock(session, lsm_tree);
-	locked = true;
-
-	/* Alter the chunks. */
-	for (i = 0; i < lsm_tree->nchunks; i++) {
```

Comment:
Yes. We'll talk.  But the short answer is that this is all now happens in `lsm_tree_worker` as it goes through all the chunks.